### PR TITLE
Use parcel geometry from mogis lv03 for feature highlight

### PR DESCRIPTION
**Pull request by ltclm**
this pr will use the mogis parcel geometry from os_realestate (npbund version) as feature highlight source for the layers:

* ch.kantone.cadastralwebmap-farbe
* ch.swisstopo-vd.amtliche-vermessung

[Testurl](https://map.geo.admin.ch/?topic=ech&lang=de&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.kantone.cadastralwebmap-farbe,ch.swisstopo-vd.amtliche-vermessung&X=238326.78&Y=611421.08&zoom=10&layers_visibility=false,true&api_url=%2F%2Fmf-chsdi3.dev.bgdi.ch%2Fdev_ltclm_av_parcel_highlight)

Note: the coverage of os_realestate is actually not as it should be, there are big holes in the dataset, we are trying to solve this. so the tooltip is not working everywhere, p.e. Geneva:
image

ping @davidoesch @rebert  @cedricmoullet @gjn

### CODE REVIEW

**Comment by davidoesch** [comment]
i like, even if the data source from the highlight is different from the wms..

**Comment by ltclm** [comment]
the geometry source should be the same: 
https://map.geo.admin.ch/?topic=ech&lang=de&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.kantone.cadastralwebmap-farbe,ch.swisstopo-vd.amtliche-vermessung&X=239441.84&Y=612904.01&zoom=13&layers_visibility=false,true&api_url=%2F%2Fmf-chsdi3.dev.bgdi.ch%2Fdev_ltclm_av_parcel_highlight

could you provide a screenshot or a url?

**Comment by davidoesch** [comment]
" geometry from os_realestate (npbund version)" is npbund geoshop respective does it have geoshop as source and same up date interval?

**Comment by ltclm** [comment]
@davidoesch the parcel geometries of geoshop are now covering the whole perimeter,
please [test](https://map.geo.admin.ch/?topic=ech&lang=de&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.kantone.cadastralwebmap-farbe,ch.swisstopo-vd.amtliche-vermessung&X=238326.78&Y=611421.08&zoom=10&layers_visibility=false,true&api_url=%2F%2Fmf-chsdi3.dev.bgdi.ch%2Fdev_ltclm_av_parcel_highlight).
about exact data origin of os_realestate and the planned data update interval please ask @stflustop

**Comment by ltclm** [comment]
has been confirmed by data owner, we can push this to production this wednesday

File: chsdi/models/vector/mogis.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    id = Column('id', Integer, primary_key=True)
+    ak = Column('ak', Unicode)
+    the_geom = Column(Geometry2D)
+
+register('ch.kantone.cadastralwebmap-farbe', CadastralWebMap)
+
+
+class CadastralWebMapOpenData(Base, Vector):
+    __tablename__ = 'view_os_realestate_vd_opendata'
+    __table_args__ = ({'schema': 'public', 'autoload': False, 'extend_existing': True})
```
Comment (loicgasser) [inline chsdi/models/vector/mogis.py]: Do we still need the extend existing ?

**Comment by loicgasser** [comment]
Feature hilight on this layer is soooo cool. LGTM. One minor comment

**Comment by ltclm** [comment]
done, 
you're right.

**Pull request merged**